compiler/src/iree/compiler/Dialect/Util/IR/UtilTypes.cpp

```diff
@@ -157,6 +157,14 @@ LogicalResult detail::verifyGlobalOp(IREE::Util::GlobalOpInterface globalOp) {
 
 LogicalResult detail::verifyGlobalAddressOp(
     GlobalAddressOpInterface addressOp, SymbolTableCollection &symbolTable) {
+  if (!isa_and_nonnull<IREE::Util::GlobalOpInterface>(
```

Comment:
Why are changes to UtilTypes necessary ?

Reply:
> Why are changes to UtilTypes necessary ?

It was not my change. I messed up with the commits, and accidentally merged two commits into mine. Anyway, I fixed it.